db/models.py

```diff
@@ -46,3 +54,71 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True)
```

Comment:
According to the tack you must set auto_now_add for created_at field to fill this field automatically during the creation